gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -459,22 +459,12 @@ gst_tensor_filter_common_set_property (GstTensorFilterPrivate * priv,
     case PROP_MODEL:
     {
       const gchar *model_files = g_value_get_string (value);
-      int idx;
 
       g_assert (model_files);
       gst_tensor_filter_parse_modelpaths_string (prop, model_files);
 
-      /* 'custom-easy' framework has a virtual model file */
-      if (g_strcmp0 (prop->fwname, "custom-easy") != 0) {
-        for (idx = 0; idx < prop->num_models; idx++) {
-          if (!g_file_test (prop->model_files[idx], G_FILE_TEST_IS_REGULAR)) {
-            g_critical ("Cannot find the model file [%d]: %s\n",
-                idx, prop->model_files[idx]);
-          }
-        }
-      }
```

Comment:
We were checking file existences if it is not "custom-easy".
It appears that you are disabling it for other subplugins as well.
Please turn them on for subplugins that really use real files as models.

Reply:
IMO, to resolve this issue, it's required to set `check_model_path_in_nns = FALSE` for "custom-easy" framework.
In this manner, we can handle all frameworks using virtual files using `check_model_path_in_nns = FALSE`

Reply:
Because except for custom-easy, they need to check model_path, they should set it TRUE, but @wooksong set them all FALSE.

Anyway, why don't we change ```check_model_path_in_nns``` to ```virtual_model_names```, which skips path checking if it is TRUE and set TRUE for custom-easy. Then, we can keep checking paths by default.

Reply:
Ahh.. understood. I also agree with your opinion.

Reply:
> virtual_model_paths

There are three cases for model checking:
1. virtual model path
2. normal case (which could be simply handled by checking in NNS)
3. I-want-to-check-these-in-my-plugin case (For example, openvino requires two model files, A.xml and A.bin. For the convenience, I make the model properties can take 1) A.xml and A.bin
2) A.xml and B.bin 3) just A)

This is why I am using the name, check_model_path_in_nns, in my PR.

In the first PR, I just set check_model_path_in_nns to false for all the sub-plugins (to keep the existing behavior). And then, check_model_path_in_nns for each sub-plugin is set to TRUE one by one after little investigating the sub-plugin source code.

Reply:
To keep the existing behavior, you need to check files for all subplugins except for custom-easy.

Reply:
And if that case, ```veify_model_paths``` should be enough. (if_necessary is not necessary)

Reply:
> To keep the existing behavior, you need to check files for all subplugins except for custom-easy.

Currently, all sub plugins have their mechanisms.

Reply:
> And, `verify_model_paths` should be enough. (if_necessary is not necessary)

Oh, that is good idea.